### PR TITLE
Corrected Jenkins repository

### PR DESCRIPTION
**Pull request by radtriste**
Switching as http is no more supported with Maven 3.8.1+
https://maven.apache.org/docs/3.8.1/release-notes.html

### CODE REVIEW

**Comment by radtriste** [comment]
tests finally ok ...

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,12 @@
         <repository>
             <id>jenkins-releases</id>
             <name>Jenkins Releases</name>
-            <url>http://repo.jenkins-ci.org/releases</url>
+            <url>https://repo.jenkins-ci.org/releases</url>
+        </repository>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
```
Comment (winklerm) [inline pom.xml:18]: Just a nitpick :) I would suggest to use the same form of the `<id>` as in `jenkins-releases`:
```suggestion
            <id>jenkins-public</id>
```

**Review by winklerm — COMMENTED** [review]
Looks good, only one naming suggestion.

File: pom.xml (hunk 1 above)
Comment (radtriste) [reply pom.xml:18]: In fact this is to correct the URL of a dependency repository definition, I will add a comment

Comment (radtriste) [reply pom.xml:19]: added the comment

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**